Exercise_D_E/assets/style/index.css

```diff
@@ -2,3 +2,125 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+body{
+    margin: 0px;
+    padding: 0px;
+}
+.icon{
+    /*z-index: 1;
+    position: absolute;*/
+    height: fit-content;
+    width: 100%;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    background-color: white;
+    border: 2px solid black;
+}
+.button{
+
+    height: fit-content;
+    width: fit-content;
+    text-align: center;
+    font-weight: bold;
+    color: white;
+    border-radius: 20px;
+    background-color: greenyellow;
+    padding: 8px;
+}
+.text{
+    font-size: xx-large;
+    height: fit-content;
+    width: fit-content;
+    text-align: center;
+    text-shadow: 2px 3px 2px black ;
+    color: rgb(255, 255, 255);
+  
+  
+    padding: 20px;
+}
+.container1{
+    height: 100%;
+    /*opacity: .6;*/
+    top: 0px;
+    position: relative;
+    background-image:linear-gradient(0deg, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),url(../images/flowershop.jpg);
+
+    /*box-shadow: inset 0 0 0 1000px rgba(0,0,0,.7);*/
+    display: flex;
+    flex-direction: column;
+    background-size: cover;
+    background-repeat: space;
+    align-items: center;
+    padding-top: 30%;
+    padding-bottom: 15%;
+ 
+    
+
+}
+
+.container2{
+    height: 400px;
+    
+    bottom: 0px;
+    font-size: x-large;
+    background-color:rgb(206, 236, 201);
+
+    /*box-shadow: inset 0 0 0 1000px rgba(0,0,0,.7);*/
+    display: flex;
+    order: 4;
+    /*lex-wrap: wrap;*/
+   
+    align-self: center;
+    
+   
+}
+.subcontainer1{
```

Comment:
I would suggest changing this to this 
`
display: float;
    position: relative;
    left: 18%;
    right: 50%;
    margin-left: 100px;
    margin-right: 200px;
`